orbit/pkg/packaging/linux_shared.go

```diff
@@ -165,7 +166,7 @@ func buildNFPM(opt Options, pkger nfpm.Packager) (string, error) {
 	contents := files.Contents{
 		&files.Content{
 			Source:      filepath.Join(rootDir, "**"),
-			Destination: "/",
+			Destination: "",
```

Comment:
See https://github.com/goreleaser/nfpm/issues/671 and https://github.com/goreleaser/nfpm/pull/576.